### PR TITLE
Content: Scaffolding - Added form working example

### PR DESCRIPTION
**Pull request by duboisp**
Added the new section under "site global" for our common widely used component, pattern and CSS.
Added all forms HTML related element as plain working example.

**Background:** This PR add the first scaffolding (or a few) working example that show how our theme style is able render plain HTML related elements. Through our accessibility upgrade project, those pages are going to be used to evaluate the WCAG 2.1 conformance of our default style applied on plain HTML element.

### CODE REVIEW

File: index-fr.md [hunk 1]
```diff
@@ -295,12 +295,18 @@ css:
 </div>
 <div class="container">
 	<h2 id="sitesglobal">Fonctionnalités globales et de sites</h2>
+	<ul>
+	  <li><a href="#composants-primaire-canadaca">Composants primaire pour le thème de Canada.ca</a></li>
```
Comment (GormFrank) [inline index-fr.md:299]:
```suggestion
	  <li><a href="#composants-primaire-canadaca">Composantes primaires pour le thème de Canada.ca</a></li>
```

File: index-fr.md [hunk 2]
```diff
@@ -295,12 +295,18 @@ css:
...
+	  <li><a href="#composants-commun">Composants commun</a></li>
```
Comment (GormFrank) [inline index-fr.md:300]:
```suggestion
	  <li><a href="#composants-commun">Composantes communes</a></li>
```

File: index-fr.md [hunk 3]
```diff
@@ -295,12 +295,18 @@ css:
...
+	</ul>
+
+	<h3 id="composants-primaire-canadaca">Composants primaire pour le thème de Canada.ca</h3>
```
Comment (GormFrank) [inline index-fr.md:303]:
```suggestion
	<h3 id="composants-primaire-canadaca">Composantes primaires pour le thème de Canada.ca</h3>
```

File: index-fr.md [hunk 4]
```diff
@@ -381,6 +387,94 @@ css:
 		</li>
 	{% endfor %}
 	</ul>
+
+	<h3 id="composants-commun">Composants commun</h3>
```
Comment (GormFrank) [inline index-fr.md:391]:
```suggestion
	<h3 id="composants-commun">Composantes communes</h3>
```

File: index-en.md [hunk 5]
```diff
@@ -295,12 +295,18 @@ css:
 </div>
 <div class="container">
 	<h2 id="sitesglobal">Sites and global functionality</h2>
+	<ul>
+	  <li><a href="#core-canadaca-theme">Core components for the Canada.ca themes</a></li>
```
Comment (GormFrank) [inline index-en.md:299]:
```suggestion
	  <li><a href="#core-canadaca-theme">Core components for the Canada.ca theme</a></li>
```

File: index-en.md [hunk 6]
```diff
@@ -295,12 +295,18 @@ css:
...
+	  <li><a href="#common-components">Common components</a></li>
+	</ul>
+
+	<h3 id="core-canadaca-theme">Core components for the Canada.ca themes</h3>
```
Comment (GormFrank) [inline index-en.md:303]:
```suggestion
	<h3 id="core-canadaca-theme">Core components for the Canada.ca theme</h3>
```

**Review by GormFrank — CHANGES_REQUESTED** [review]

File: common/scaffolding/forms.html [hunk 7]
```diff
@@ -0,0 +1,1150 @@
...
+<fieldset>
+	<legend>Display</legend>
+	<p><label><input type=radio name=c value=0 checked> Black on White</label>
+	<p><label><input type=radio name=c value=1> White on Black</label>
+	<p><label><input type=checkbox name=g> Use grayscale</label>
+	<p><label>Enhance contrast <input type=range name=e list=contrast min=0 max=100 value=0 step=1></label>
+	<datalist id=contrast>
+		<option label=Normal value=0>
+		<option label=Middle value=50>
+		<option label=Maximum value=100>
```
[the remark this answers is not in the document]
Comment (duboisp) [reply common/scaffolding/forms.html:1069]: I took the working example mostly as is from the HTML5 specification

File: common/scaffolding/forms.html [hunk 8]
```diff
@@ -0,0 +1,1150 @@
...
+		&lt;option label=Maximum value=100>
+	&lt;/datalist>
+&lt;/fieldset>
+&lt;/form></code></pre>
+
+<h3>Disabled fieldset</h3>
+<form>
+<fieldset name="clubfields" disabled>
+	<legend> <label>
+		<input type=checkbox name=club onchange="form.clubfields.disabled = !checked">
```
[the remark this answers is not in the document]
Comment (duboisp) [reply common/scaffolding/forms.html:1092]: I took the working example mostly as is from the HTML5 specification

File: common/scaffolding/forms.html [hunk 9]
```diff
@@ -0,0 +1,1150 @@
...
+	<form>
+		<label for="input-email">Label: </label><input id="input-email" type="email">
+	</form>
+	<pre><code>&lt;form>
+	&lt;label for="input-email">Label: &lt;/label>&lt;input id="input-email" type="email">
+&lt;/form></code></pre>
+	</div>
+</div>
+
+<h4>With the input mode email (<code>inputmode=email</code>)</h4>
```
[the remark this answers is not in the document]
Comment (duboisp) [reply common/scaffolding/forms.html:211]: I don't think so, A visual keyboard can be available on any touch device or turn on for our desktop.

File: common/scaffolding/forms.html [hunk 10]
```diff
@@ -0,0 +1,1150 @@
...
+
+
+<h2>The <code>fieldset</code> and <code>legend</code> element</h2>
+<form>
+<fieldset>
+	<legend>Display</legend>
+	<p><label><input type=radio name=c value=0 checked> Black on White</label>
+	<p><label><input type=radio name=c value=1> White on Black</label>
+	<p><label><input type=checkbox name=g> Use grayscale</label>
+	<p><label>Enhance contrast <input type=range name=e list=contrast min=0 max=100 value=0 step=1></label>
```
[the remark this answers is not in the document]
Comment (duboisp) [reply common/scaffolding/forms.html:1065]: As discussed

```html
<p><label>Enhance contrast <input type=range name=e list=contrast min=0 max=100 value=0 step=1></label>

<option label="Middle High" value="78">
		</option>
```

File: common/scaffolding/forms.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (duboisp) [reply common/scaffolding/forms.html:1092]: Let remind us to include that behaviour into our accessibility report.

**Review by GormFrank — APPROVED** [review]

**Pull request merged**